packages/components/src/slot-fill/bubbles-virtually/slot-fill-context.js

```diff
@@ -4,6 +4,7 @@
 import { createContext } from '@wordpress/element';
 
 const SlotFillContext = createContext( {
+	__unstableDefaultContext: true,
 	slots: {},
 	fills: {},
 	registerSlot: () => {},
```

Comment:
Could we warn instead when the default function that registers slots is called? This way we would avoid introducing the unstable property and the warning would fire in almost the same place.

I’m suggesting moving the warning call to registerSlot. What do you think?

Reply:
This makes a lot more sense. Will make the change.